test/specs/scenarios/document.js

```diff
@@ -0,0 +1,132 @@
+import { describe, it } from '../../utils/mochaw'
+import { localhostUrl } from '../../utils/config'
+import { goToPassportUploadScreen, uploadFileAndClickConfirmButton } from './sharedFlows.js'
+
+export const documentScenarios = (driver, screens, lang) => {
+  const {
```

Comment:
In some files, we have these `const`s in one line in others one below another. Shall we keep the formatting consistent?

Reply:
I think it's ok to stay on 1 line if it's just a couple of variables. It's when there's a lot like in this case that I find it better for it to be broken down into new lines otherwise it's hard to see all the variables.

Reply:
I agree with @kopijunkie, I think I misread the original comment. I thought you meant that there was an inconsistence in where these variables were being defined within the files.